flickit-assessment-core/src/main/java/org/flickit/assessment/core/adapter/in/rest/confidencelevel/CalculateConfidenceLevelRestController.java

```diff
@@ -0,0 +1,36 @@
+package org.flickit.assessment.core.adapter.in.rest.confidencelevel;
+
+import lombok.RequiredArgsConstructor;
+import org.flickit.assessment.core.application.port.in.confidencelevel.CalculateConfidenceLevelUseCase;
+import org.flickit.assessment.core.application.port.in.confidencelevel.CalculateConfidenceLevelUseCase.Result;
+import org.flickit.assessment.core.application.port.in.confidencelevel.CalculateConfidenceLevelUseCase.Param;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+public class CalculateConfidenceLevelRestController {
+
+    private final CalculateConfidenceLevelUseCase useCase;
+
+    @PostMapping("/assessments/{assessmentId}/calculate-confidence-level")
```

Comment:
change url to `@PostMapping("/assessments/{assessmentId}/calculate-confidence")`